src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/inputs/EceWorkExperienceReferencePreview.vue

```diff
@@ -66,11 +66,18 @@ export default defineComponent({
   },
   setup: () => {
     const wizardStore = useWizardStore();
+    const applicationStore = useApplicationStore();
 
     return {
-      wizardStore,
+      wizardStore,applicationStore,
     };
   },
+  methods:{
+    setWizard(stage:string) {
```

Comment:
should this be set to a PortalStage type instead of string? https://github.com/bcgov/ECC-ECER/blob/41d7abce57ecf88940013690c1b8803b52ebad12/src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/types/openapi.d.ts#L110